dist/mixins/pagination/base/pagination-mixins.less

```diff
@@ -1,14 +0,0 @@
-.pagination-layout-base() {
```

Comment:
I see there's a few unrelated mixin files deleted. Just checking this is intentional...

Reply:
I think those src files were deleted but when I reran the build, those got removed then. 
I don't see any src files but only the dist ones. Should those be there even if there is not src version of them?

Reply:
Ah, ok. Sounds good. Thanks. Will sign off.